src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureColor {
```

Comment:
The name 'FigureColor' sounds like it is only related to color, which is not suitable for a class that 'IsoscelesTrapezoid' would extend from. Consider using a more appropriate name or design.